wiki/Tournaments/TWC/2021/en.md

```diff
@@ -31,14 +31,17 @@ The **osu!taiko World Cup 2021** (***TWC 2021***) is a country-based osu!taiko t
 | ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | $80 per team member, unique profile badge |
 | ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | $40 per team member, unique profile badge |
 
+![image](https://user-images.githubusercontent.com/30021847/117518638-7b181e00-af98-11eb-8771-32a243ea78f1.png "TWC 2021 winner badge") ![image](https://user-images.githubusercontent.com/30021847/117518656-8ff4b180-af98-11eb-98e5-541283fca1bc.png "TWC 2021 2nd Place badge")
+![image](https://user-images.githubusercontent.com/30021847/117518661-97b45600-af98-11eb-9c4a-285d05fecef6.png "TWC 2021 3rd Place badge")
```

Comment:
I don't think this is an appropriate way of inserting images. You need to upload them to a folder named `img` under `TWC/2021` and then replace these URLs with `/wiki/Tournaments/TWC/2021/img/<filename>.png`

Reply:
fixed